cli/embedded-app/src/main.ts

```diff
@@ -0,0 +1,16 @@
+import Vue from 'vue';
+import App from './App.vue';
+import router from './router';
+import store from './store';
+
+import { registerCustomElements } from '../../../dist/index.js';
```

Comment:
It would be nice to have this act like documentation and be imported from '@purecloud/iframe-coordinator' (or whatever the package name is).  But I am not certain a way to do that without adding complexity of lerna or npm links...

Reply:
I tried that initially, but ran into some issues with babel trying to transpile the already transpiled module. I do have a better understanding of what was going on there now though, so maybe I can get the previous version working again. I'll give it a shot.